ngraph_bridge/ngraph_encapsulate_impl.h

```diff
@@ -0,0 +1,208 @@
+/*******************************************************************************
+ * Copyright 2017-2019 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+#pragma once
```

Comment:
Not sure if both `#pragma once` and `header guard` is required. But, I did read somewhere that if you have both the `#pragma once` should come after the `header guard`.

Reply:
@kanvi-nervana Could you please find out the source? It would be a good practice to follow.